soda/contracts/design_decisions.md

```diff
@@ -0,0 +1,52 @@
+# Soda data contract YAML format
+
+### No data source & connection details
+
+We leave datasource, connection and schema out of the contract files
+  * Pro
+    * The same contract file can be used to run in CI/CD
+    * Leveraging the existing Soda configuration files
+  * Con: 
+    * more CLI args
```

Comment:
this can easily be addressed with a project config file where defaults could be held for example.